superset-frontend/src/SqlLab/components/QueryTable/index.tsx

```diff
@@ -187,16 +191,14 @@ const QueryTable = ({
         const status = statusAttributes[state] || statusAttributes.error;
 
         if (q.endDttm) {
-          q.duration = fDuration(q.startDttm, q.endDttm);
+          q.duration = (
+            <Label
+              style={{ 'font-family': theme.typography.families.monospace }}
+            >
```

Comment:
Since we're doing this twice here with style overrides, it might make sense to build it into the Label component as a "monospace" prop.

Reply:
I agree, was on the edge of doing it there. Can refactor this in